libs/application/data-providers/src/providers/QualityPhotoProvider.ts

```diff
@@ -3,18 +3,19 @@ import {
   SuccessfulDataProviderResult,
   FailedDataProviderResult,
   Application,
+  coreErrorMessages,
 } from '@island.is/application/core'
-import { DrivingLicenseFakeData, YES } from '../lib/constants'
-import { m } from '../lib/messages'
+import { DataProviderFakeData, YES } from '../libs/constants'
 
 export class QualityPhotoProvider extends BasicDataProvider {
   type = 'QualityPhotoProvider'
 
   async provide(application: Application) {
     const fakeData = application.answers.fakeData as
-      | DrivingLicenseFakeData
+      | DataProviderFakeData
       | undefined
 
+    // To use fake data for the quality photo provider take a look at the implementation in libs/application/templates/driving-license/src/forms/application.ts
     if (fakeData?.useFakeData === YES) {
```

Comment:
Would be good to make sure that `useFakeData === YES` does not work on production.

Reply:
This should be addressed now